egs/wsj/s5/steps/nnet3/chain/train.py

```diff
@@ -280,6 +284,9 @@ def ProcessArgs(args):
     if args.chunk_right_context < 0:
         raise Exception("--egs.chunk-right-context should be non-negative")
 
+    if args.ivector_interval < 0 or args.ivector_interval > args.chunk_width:
+        raise Exception("--egs.ivector-interval should be non-negative and should not be greater than egs.chunk-width")
```

Comment:
I think there's no reason to forbid ivector-interval > chunk-width-- it doesn't make sense but it's harmless and no need to introduce extra code to check for it.